test/functional/ui/sign_spec.lua

```diff
@@ -239,7 +239,7 @@ describe('Signs', function()
         {4:                                                     }|
         :sign place                                          |
         {9:--- Signs ---}                                        |
-        {10:Signs for [NULL]:}                                    |
+        {10:Signs for (null):}                                    |
```

Comment:
Why did this happen?

Reply:
Nevermind. Other tests failed because `vim_vsnprintf` and `vim_vsnprintf_typval` behave differently from `vsnprintf`.

Reply:
An alternative is to print the buffer number if the buffer doesn't have a name. I don't think it was intended to print the internal representation of a null pointer to start with...